langgraph/src/pregel/write.ts

```diff
@@ -19,19 +20,19 @@ export class ChannelWrite<
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   RunInput = any
 > extends RunnablePassthrough<RunInput> {
-  channels: Array<ChannelWriteEntry>;
+  writes: Array<ChannelWriteEntry>;
 
-  constructor(channels: Array<ChannelWriteEntry>) {
-    const name = `ChannelWrite<${channels
+  constructor(writes: Array<ChannelWriteEntry>) {
+    const name = `ChannelWrite<${writes
       .map(({ channel }) => channel)
       .join(",")}>`;
     super({
-      ...{ channels, name },
+      ...{ channels: writes, name },
```

Comment:
this should be channels in super call too?

Reply:
Unexpected change by VSCode when applying the `Rename Symbol` action. I changed to `...{ writes, name }` and everything seems to work the same.